tools/llm_bench/task/image_generation.py

```diff
@@ -120,6 +192,11 @@ def run_image_generation_benchmark(model_path, framework, device, args, num_iter
         raise RuntimeError('==Failure prompts is empty ==')
     log.info(f'Benchmarking iter nums(exclude warm-up): {num_iters}, prompt nums: {len(image_list)}, prompt idx: {prompt_idx_list}')
 
+    if not use_genai:
+        text_gen_fn = run_image_generation
```

Comment:
very misleading naming....
```suggestion
        image_gen_fn = run_image_generation
```